css/src/components/index.scss

```diff
@@ -0,0 +1 @@
+@import '';
```

Comment:
Can the file remain empty while satisfying lint? If not, we can remove it and add a README.md

Reply:
Readme could be populated with some of details from the plan: 
https://github.com/microsoft/atlas-design/pull/2

Reply:
Current lint process not allowing the file to remain empty . So removed it